include/ceed/types.h

```diff
@@ -7,7 +7,8 @@
 
 /// @file
 /// Public header for types and macros used in user QFunction source code
-#pragma once
+#ifndef CEED_QFUNCTION_DEFS_H
```

Comment:
Small note - I've seen some people recommend both. Not sure I fully understand the argument why though

Reply:
Yeah, both is a bit perplexing... Maybe to avoid possible guard name conflicts?

Reply:
I think for a while `#pragma once` gave faster compile times, but nowadays that's less of a factor